packages/components/src/components/Button/index.ts

```diff
@@ -20,6 +20,9 @@ export const ButtonComponentDefinition: ComponentDefinition = {
     'cfLineHeight',
     'cfBorder',
   ],
+  tooltip: {
+    description: 'A button that can be used to trigger an action or navigate to a different page',
+  },
```

Comment:
super nit: seems like we add a period to all of the other descriptions, should we add here?